omnigibson/robots/manipulation_robot.py

```diff
@@ -1101,8 +1169,10 @@ def _handle_assisted_grasping(self, action):
             controller = self._controllers[f"gripper_{arm}"]
             controlled_joints = controller.dof_idx
             threshold = np.mean(np.array(self.control_limits["position"])[:, controlled_joints], axis=0)
-            applying_grasp = np.any(controller.control < threshold)
-
+            if self._grasping_direction == "lower":
+                applying_grasp = np.any(controller.control < threshold)
+            else:
+                applying_grasp = np.any(controller.control > threshold)
```

Comment:
what about == case?

Reply:
I think we are only checking grasp here, so == means no grasp for both conditions